src/style.css

```diff
@@ -0,0 +1,65 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.moyo__logo {
+  height: 40;
+  width: 40;
+}
+
+.nav__menu {
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  list-style: none;
+}
+
+.nav__link {
+  text-decoration: none;
+  position: relative;
+  display: inline-block;
+  padding: 23px 0;
```

Comment:
make your line-height equal to the height of your container, instead of using padding

Reply:
Fixed) Thanks